Makefile

```diff
@@ -2,6 +2,16 @@ SUBDIRS = $(wildcard [0-9].*)
 .PHONY: $(SUBDIRS)
 all: $(SUBDIRS)
 $(SUBDIRS):
-	cat Dockerfile  | sed 's/^FROM.*/FROM php:$@-alpine/' > $@/Dockerfile
+	@[ -f $@/Dockerfile ] || cat Dockerfile.template  | sed 's/^FROM.*/FROM php:$@-alpine/' > $@/Dockerfile | echo "Created $@/Dockerfile from Dockerfile.template"
 
 
+clean:
+	rm ?.?/Dockerfile
+
+build: clean all
+	(cd 7.2; docker build . -t atk4/image:7.2)
+	(cd 7.3; docker build . -t atk4/image:7.3)
+	(cd 8.0.0alpha1; docker build . -t atk4/image:7.3)
```

Comment:
prefer some loop, the 3rd one is even wrong I think (no alpha, why 7.3 tag twice, ..)

Reply:
fixed. Makefile syntax is pretty bad, no loop, at least i'm not familiar with loops in makefiles. Added comments.